src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -507,9 +507,10 @@ protected void DisposeAllFailedNodesTimers()
         public void Execute<TResult>(
             RavenCommand<TResult> command,
             JsonOperationContext context,
-            SessionInfo sessionInfo = null)
+            SessionInfo sessionInfo = null,
+            CancellationToken token = default)
```

Comment:
Not a good idea for public method in request executor. Use async one or write a helper method in replication loader.